app/src/pages/inside/filtersPage/filterGrid/filterName/filterName.jsx

```diff
@@ -86,6 +86,7 @@ export class FilterName extends Component {
       nameLink,
     } = this.props;
 
+    // eslint-disable-next-line react/prop-types
```

Comment:
I suppose it is better to not create components during render.
This component can be simply replaced with direct condition usage inside of `return`.